tools/scan_symbols.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
This doesn't work for me - Ubuntu doesn't interpret `python` as `python3`

FWIW other scripts are using:

```suggestion
#! /usr/bin/python3
```
And that does work

Reply:
Huh, my 21.10 system does. I wonder if I explicitly set that up at some point.

Reply:
21.10 is EOL!